docs/family-pub-key.rst

```diff
@@ -0,0 +1,101 @@
+******************************
+Public Key Transaction Family
+******************************
+
+Overview
+========
+
+The *Public Key* transaction family provides a trustless way to store the public key based on RSA cryptosystem on the REMchain.
+
+Definitions of Public Key Entries
+---------------------------------
+
+The following protocol buffers definition define public key entries:
+
+.. code-block:: protobuf
+
+    message PubKeyStorage {
+         // The UID field of a certificate.
```

Comment:
Actually the public key of a user who posted the key. The entity behind that public key is necessarily a X.509 certificate.

Reply:
Isn't this pub key of a user already available in Transaction Handler?